Makefile.am

```diff
@@ -68,9 +68,14 @@ release:
 	echo "* $$(LC_TIME='en_US.UTF-8' date +'%a %b %d %Y') $$(git config --get user.name) <$$(git config --get user.email)> $$NEW_VER-1" | sort > /tmp/changelog.tmp; \
 	git log --oneline $$OLD_VER..HEAD | awk '{$$1=""; if (a[$$0]++ == 0) print "-" $$0} END {print ""}' | grep -v -e "- Merge" -e "- testsuite:" -e "- make:" >> /tmp/changelog.tmp; \
 	sed "$$(grep -n %changelog libreport.spec.in | cut -f1 -d: | head -1)"'r /tmp/changelog.tmp' -i libreport.spec.in; \
-	sed -e "s/^## \[Unreleased\]/## [Unreleased]\n\n## [$$NEW_VER] - $$(date +'%F')/" \
+	sed -e "s/^## \[Unreleased\]/## [Unreleased]\n\n## [$$NEW_VER] - $$(date +'%F')\n[\/\/] # (Please sum up added, changed, removed and fixed features here. See below for examples.)/" \
 	    -e "s/^\[Unreleased\]: \(https:\/\/.*\/compare\)\(\/.*\)\.\.\.HEAD/[Unreleased]: \1\/$$NEW_VER...HEAD\n[$$NEW_VER]: \1\2...$$NEW_VER/" \
-	    -i CHANGELOG.md; \
+	    -i CHANGELOG.md
+	if [ -z $$EDITOR ]; then \
+		/usr/bin/vi CHANGELOG.md; \
+	else \
+		$(EDITOR) CHANGELOG.md; \
```

Comment:
You can replace this with a one-liner: `${EDITOR:-vi} CHANGELOG.md`.

https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02

Reply:
For completeness, `VISUAL` should also be consulted, but ehhhhhhhhhhhhhhhhhhhhhhhhhh.

Reply:
> You can replace this with a one-liner  
  
Wow, nice, I had no idea this exists. Replaced.